### ISSUE TITLE
Compare xgboost with TFBT

### ISSUE THREAD

**Issue by pseudotensor** [h2oai/h2o4gpu#286]
@arno, @rory @jon could you please have a look:
TF Boosted Trees (TFBT) is a new open-sourced frame-work for the distributed training of gradient boosted trees. It is based on TensorFlow, and its distinguishing features include a novel architecture, automatic loss differentiation, layer-by-layer boosting that results in smaller ensembles and faster prediction, principled multi-class handling, and a number of regularization techniques to prevent overfitting.
https://arxiv.org/abs/1710.11555
GitHub: https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/boosted_trees (edited)
